contracts/Rewards/RewardsDistributor.sol

```diff
@@ -405,32 +401,21 @@ contract RewardsDistributor is ExponentialNoError, Ownable2StepUpgradeable {
      * @notice Claim all rewardToken accrued by the holders
```

Comment:
```suggestion
     * @notice Claim all rewardToken accrued by the holders.
```

Reply:
Fixed , can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/151/commits/ffd493f85fd85b0e1ede68a5b5b8ca0e298698e4).